src/bin/pg_dump/pg_dump_sort.c

```diff
@@ -988,6 +988,28 @@ repairDependencyLoop(DumpableObject **loop,
 		return;
 	}
 
+	/*
+	 * A multi-statement TVF will have dependency loop with it's
```

Comment:
It would be safer if we can apply the below logics to only pltsql functions.

Reply:
Yup, moved the logic to handle this cyclic dependency in `bbf_fixTableTypeDependency` function.